src/main/scala/org/scalajs/dom/raw/lib.scala

```diff
@@ -975,8 +975,52 @@ abstract class Element
    */
   def removeAttributeNS(namespaceURI: String,
       localName: String): Unit = js.native
+
+  /**
+   * The Element.requestFullscreen() method issues an asynchronous request to
+   * make the element be displayed in full-screen mode.
+   *
+   * It's not guaranteed that the element will be put into full screen mode.
+   * If permission to enter full screen mode is granted, the returned Promise
+   * will resolve and the element will receive a fullscreenchange event to let
+   * it know that it's now in full screen mode. If permission is denied, the
+   * promise is rejected and the element receives a fullscreenerror event
+   * instead. If the element has been detached from the original document, then
+   * the document receives these events instead.
+   *
+   * Earlier implementations of the Fullscreen API would always send these
+   * events to the document rather than the element, and you may need to be
+   * able to handle that situation. Check Browser compatibility in fullscreen
+   * for specifics on when each browser made this change.
+   *
+   * MDN
+   */
+  def requestFullscreen(
+      options: FullscreenOptions = ???): js.Promise[Unit] = js.native
```

Comment:
Prefer `js.native` instead of `???` for default values in JS APIs:
```suggestion
      options: FullscreenOptions = js.native): js.Promise[Unit] = js.native
```

Reply:
Fixed in https://github.com/scala-js/scala-js-dom/pull/402/commits/075d58a381576ca63f41205c6cccf238cddffc19